applications/nrf5340_audio/unicast_server/main.c

```diff
@@ -468,19 +469,16 @@ void streamctrl_send(void const *const data, size_t size, uint8_t num_ch)
 	}
 }
 
-int streamctrl_start(void)
+int main(void)
 {
 	int ret;
-	static bool started;
 	struct bt_data ext_adv_buf[CONFIG_EXT_ADV_BUF_MAX];
```

Comment:
No `LOG_DBG("nRF5340 APP core started");` would be nice to have for consistency.